INSTALLATION.md

```diff
@@ -0,0 +1,48 @@
+## Development ##
+There are several methods for building and setting up a development environment.
+  
+### Manual Method ###
+1. Create a new virtualenv with Python 3.6 (`virtualenv --python=<path-to-py3> venv`) and source it
+2. Install requirements via pip. For the moment there are two options:
+   * Install with plain pip using the `requirements`
+   * Install using Pipenv (which reads from the Pipfile)
+3. Build nativepython libraries using `python setup.py build`
+4. Append the root of this repository to your `PYTHONPATH`
+  
+### Pipenv Method ###
+1. (Optional) Create a new virtualenv with Python 3.6 (`virtualenv --python=<path-to-py3> venv`) and source it. If you choose to use Pipenv alone, it will create an appropriate virtualenv for you.
```

Comment:
Since the `Makefile` implements this workflow, should we document how to use that as well or instead of these steps? Right now, `make install` should perform steps 1 and 2 of these instructions. Feel free to change the default python parameter of the `Makefile` from `python3` to `python3.6` on line 5

Reply:
Yes, I will add a section for the Makefile methods and note a caveat about the python versions